source-sqlserver/datatypes.go

```diff
@@ -143,6 +147,15 @@ func (db *sqlserverDatabase) translateRecordField(columnType interface{}, val in
 			}
 			return u.String(), nil
 		}
+		if len(val) > truncateColumnThreshold {
+			val = val[:truncateColumnThreshold]
+		}
+		return val, nil
+	case string:
+		if len(val) > truncateColumnThreshold {
+			val = val[:truncateColumnThreshold]
```

Comment:
Is it possible / would it be a problem for this to split a multi-byte character?

Reply:
Ah yeah that makes sense. Since the JSON serialization won't produce invalid UTF-8 (I suppose it can't, by definition) then it doesn't seem like we need to worry about this anymore.